sdk/tests/conformance2/rendering/blitframebuffer-multisampled-readbuffer.html

```diff
@@ -126,35 +93,13 @@
 }
 
 function checkPixel(color, expectedColor) {
-  var tolerance = 7;
+  var tolerance = 1;
```

Comment:
This seems pretty tight to me, maybe 3 or 5? which is enough to catch any bugs in the functionality.